commands/create_package.go

```diff
@@ -52,6 +52,9 @@ func CreatePackage(logger logging.Logger, client PackClient) *cobra.Command {
 	cmd.MarkFlagRequired("package-config")
 	cmd.Flags().BoolVar(&flags.Publish, "publish", false, "Publish to registry")
 	AddHelpFlag(cmd, "create-package")
+	
+	cmd.Hidden = true
```

Comment:
This will also have the effect of disabling the acceptance tests, since it will no longer show up in the help output. I'm okay with that if everyone else is.